src/neuroglancer/datasource/computed/tensorflow/frontend.ts

```diff
@@ -0,0 +1,278 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// import * as tfjs from '@tensorflow/tfjs';
+import {ComputedVolumeDataSourceParameters, VolumeComputationFrontend, VolumeComputationFrontendProvider} from 'neuroglancer/datasource/computed/frontend';
+import {InferenceRequest, InferenceResult, TENSORFLOW_COMPUTATION_RPC_ID, TENSORFLOW_INFERENCE_RPC_ID, TensorflowArray, TensorflowComputationParameters} from 'neuroglancer/datasource/computed/tensorflow/base';
+import {DataType, VolumeType} from 'neuroglancer/sliceview/volume/base';
+import {CANCELED, CancellationToken} from 'neuroglancer/util/cancellation';
+import {verifyFloat, verifyString} from 'neuroglancer/util/json';
+import {registerPromiseRPC, registerSharedObjectOwner, RPCPromise} from 'neuroglancer/worker_rpc';
+
+let tfjs: any = null;
```

Comment:
I think you can make this properly typed in various ways.  E.g. I think typeof import('@tensorflow/tfjs') works.

Reply:
Done.